src/libraries/System.ComponentModel.TypeConverter/tests/System.ComponentModel.TypeConverter.Tests.csproj

```diff
@@ -153,6 +153,8 @@
     <Compile Include="Security\Authentication\ExtendedProtection\ExtendedProtectionPolicyTypeConverterTests.cs" />
     <Compile Include="XTypeDescriptionProviderTests.cs" />
 
+    <Compile Include="$(CommonTestPath)TestUtilities\System\DisableParallelization.cs"
+        Link="Common\TestUtilities\System\DisableParallelization.cs" />
```

Comment:
Why not just include it as part of `TestUtilities.csproj` which is referenced by default on all test projects?

Reply:
> Why not just include it as part of TestUtilities.csproj which is referenced by default on all test projects?

@safern this is what I've tried in the beginning and got a lot of new CI failures.

From https://xunit.net/docs/shared-context#collection-fixture I've learned that:

> Fixtures can be shared across assemblies, but collection definitions must be in the same assembly as the test that uses them.

So we have to reference the file in every project that wants to use it ;(

Reply:
Ah i see, that's unfortunate.